index.js

```diff
@@ -55,7 +55,26 @@ function updateBuildStatus(updateConfig, callback) {
     });
 }
 
-const supportFunction = { updateBuildStatus };
+/**
+ * Shutdown workers and then exit the process
+ * @method shutDownWorker
+ * @param  {Object}       worker    worker to be ended
+ * @param  {Function}     callback  callback function
+ * @return null
+ */
+function shutDownWorker(worker, callback) {
+    worker.end((err) => {
+        if (err) {
+            winston.error(`failed to end the worker: ${err}`);
+            process.exit(128);
+        }
+
+        process.exit(0);
+        callback(null); // Only for unit test, will never get here in reality
+    });
+}
+
+const supportFunction = { updateBuildStatus, shutDownWorker };
```

Comment:
group the support functions together. And I need to mock updateBuildStatus after requiring the index. If it's a function by itself without the wrap, it will be registered when I  require the file and my mock later on won't work.